package/gluon-web-network/i18n/de.po

```diff
@@ -19,8 +19,8 @@ msgstr "Automatisch (RA/DHCPv6)"
 msgid "Disabled"
 msgstr "Deaktiviert"
 
-msgid "Enable PoE passthrough"
-msgstr "PoE-Passthrough aktivieren"
+msgid "Enable %s"
+msgstr "%s aktivieren"
```

Comment:
I don't really like that "PoE-Passthrough" won't be formatted correctly in German anymore. It would be nice to make the GPIO names translatable as well.